modules/scanning-images.adoc

```diff
@@ -121,7 +121,7 @@ Scanner identifies vulnerabilities in images that contain the following Linux di
 | Distribution | Version
 
 | link:https://www.alpinelinux.org/[Alpine Linux]
-| `alpine:3.2`^[1]^,`alpine:3.3`, `alpine:3.4`, `alpine:3.5`, `alpine:3.6`, `alpine:3.7`, `alpine:3.8`, `alpine:3.9`, `alpine:3.10`, `alpine:3.11`, `alpine:3.12`, `alpine:3.13`, `alpine:3.14`, `alpine:3.15`, `alpine:3.16`, `alpine:3.17`, `alpine:3.18`, `alpine:3.19`^[2]^, `alpine:edge`
+| `alpine:3.2`^[1]^,`alpine:3.3`, `alpine:3.4`, `alpine:3.5`, `alpine:3.6`, `alpine:3.7`, `alpine:3.8`, `alpine:3.9`, `alpine:3.10`, `alpine:3.11`, `alpine:3.12`, `alpine:3.13`, `alpine:3.14`, `alpine:3.15`, `alpine:3.16`, `alpine:3.17`, `alpine:3.18`, `alpine:3.19`^[2]^, `alpine:3.20`^[2]^, `alpine:edge`
```

Comment:
Both `alpine:3.19` and `alpine:3.20` are supported by both Scanner v2 and V4. Can you remove the `[2]`?